k9mail-library/src/main/java/com/fsck/k9/mail/Address.java

```diff
@@ -64,7 +64,7 @@ private Address(String address, String personal, boolean parse) {
                     mPersonal = (personal == null) ? null : personal.trim();
                 }
             } else {
-                // This should be an error
+                Timber.e(new IllegalArgumentException(), "address " + address + " is invalid");
```

Comment:
We don't want to create an exception object just for logging. Also, Timber supports format strings. So this could be:

```java
Timber.e("Invalid address: %s", address);
```

Reply:
Wasn't aware either was possible. Changing now, thanks for the tip.